ui/components/multichain/edit-networks-modal/edit-networks-modal.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
can we add unit tests for this new component

Reply:
For now, this is a dumb component. I will add unit tests to it when I will make it functional